cpp/src/lists/count_elements.cu

```diff
@@ -52,7 +52,7 @@ std::unique_ptr<column> count_elements(lists_column_view const& input,
   // create output column
   auto output = make_fixed_width_column(data_type{type_to_id<size_type>()},
                                         input.size(),
-                                        copy_bitmask(input.parent()),
+                                        copy_bitmask(input.parent(), stream, mr),
```

Comment:
Do we need to be explicit about `detail::` here? Once we add streams to the public API, will the signatures of public and detail be the same?

Reply:
The compiler does not have a problem with it. I'm happy to be explicit if it helps with reading the code.
Yes, once we add streams to the public API, the signatures will be the same minus the namespace.

Which brings up the question: Perhaps we should consider removing detail APIs that match public APIs in the future?

Reply:
> Perhaps we should consider removing detail APIs that match public APIs in the future?

We’ve ruled against this in past conversations because of the NVTX ranges in the public API that we don’t want at the detail level.